Jenkinsfile

```diff
@@ -117,11 +117,15 @@ try {
 
             // Prepare Docker containers used by Instrumentation tests
             // TODO: How much of this logic can be moved to start_server.sh for shared logic with local testing.
+
+            def tempDir = runCommand('mktemp -d -t app_config')
+            sh "tools/sync_test_server/app_config_generator.sh ${tempDir} tools/sync_test_server/app_template testapp1 testapp2"
+
             sh "docker network create ${dockerNetworkId}"
             mongoDbRealmContainer = mdbRealmImage.run("--network ${dockerNetworkId}")
             mongoDbRealmCommandServerContainer = commandServerEnv.run("--network container:${mongoDbRealmContainer.id}")
-            sh "docker cp tools/sync_test_server/app_config ${mongoDbRealmContainer.id}:/tmp/app_config-testapp1"
-            sh "docker cp tools/sync_test_server/app_config ${mongoDbRealmContainer.id}:/tmp/app_config-testapp2"
+            sh "docker cp ${tempDir}/testapp1 ${mongoDbRealmContainer.id}:/tmp/app_config-testapp1"
```

Comment:
This will not work out. You have to follow the same approach of `start_server.sh` by mounting the app configuration dir in both the sync-server and the command server. Unfortunately there is no blocking command like the `setup_mongodb_realm.sh` to ensure that all apps are imported, so maybe we need a loop to check that both `app_config-testapp{1,2}/app_id` files have been created (or that the command server responds with 200 on `localhost:8888/testapp{1,2}`.